lib/files/tailwind.config.js

```diff
@@ -1,16 +1,18 @@
+// Learn more at https://tailwindcss.com/docs/configuration
 module.exports = ({ dev, rootDir, srcDir }) => ({
-  theme: {},
-  variants: {},
-  plugins: [],
-  purge: {
-    // Learn more on https://tailwindcss.com/docs/controlling-file-size/#removing-unused-css
-    enabled: !dev,
```

Comment:
Why removing this flag?

Reply:
Because it is already the default behaviour.

> Now whenever you compile your CSS with NODE_ENV set to production, Tailwind will automatically purge unused styles from your CSS.

🔗 https://tailwindcss.com/docs/optimizing-for-production#removing-unused-css

Reply:
@Atinux  Any timeframes when this could be approved and released?